data/json/items/ammo/300BLK.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "300BLK_supersonic",
+    "type": "AMMO",
+    "name": ".300 AAC Blackout",
+    "description": ".300 AAC Blackout is an intermediate cartridge that achieves ballistics similar to the 7.62x39 but allows use on the AR-15 platform.  The round is necked-up from the 5.56mm NATO, but feeds from a STANAG magazine.  It requires a specific barrel, so will not work in a standard M4 or similar carbine.  This loading is supersonic.",
+    "weight": 18,
+    "volume": 1,
+    "price": 3500,
+    "material": [ "brass", "powder" ],
+    "symbol": "=",
+    "color": "yellow",
+    "count": 30,
+    "stack_size": 47,
+    "ammo_type": "300BLK",
+    "casing": "300BLK_casing",
+    "range": 45,
+    "damage": 43,
+    "pierce": 5,
+    "dispersion": 30,
+    "recoil": 2700,
```

Comment:
2700 seems high for an intermediate cartridge like this.

Reply:
Yeah, it's all guessy at this point - I'll drop to 2000.